phoenicis-dist/src/flatpak/org.phoenicis.javafx.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">org.phoenicis.javafx</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Phoenicis</name>
+  <summary>Install and run Windows software on Linux</summary>
+  <summary xml:lang="de">Installiere und starte Windows Software unter Linux</summary>
+  <description>
+    <p>Phoenicis is a front-end for Wine.</p>
+  </description>
+  <description xml:lang="de">
+    <p>Phoenicis ist ein Front-End für Wine.</p>
+  </description>
+  <categories>
+    <category>Game</category>
```

Comment:
I'm not sure if `Game` is the correct category for Phoenicis.
Maybe something like `Utility` fits better.